pkg/reconciler/v1alpha1/revision/resources/deploy_test.go

```diff
@@ -384,28 +386,18 @@ func TestMakePodSpec(t *testing.T) {
 		}),
 	}, {
 		name: "simple concurrency=single with owner",
-		rev: &v1alpha1.Revision{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace: "foo",
-				Name:      "bar",
-				UID:       "1234",
-				Labels:    labels,
-				OwnerReferences: []metav1.OwnerReference{{
+		rev: revision(
+			withContainerConcurrency(1),
+			func(revision *v1alpha1.Revision) {
+				revision.ObjectMeta.OwnerReferences = []metav1.OwnerReference{{
```

Comment:
Owner references are used twice as well, time to add an option?

Reply:
Done.